requests/sessions.py

```diff
@@ -222,9 +222,12 @@ def rebuild_proxies(self, prepared_request, proxies):
         url = prepared_request.url
         scheme = urlparse(url).scheme
         new_proxies = proxies.copy() if proxies is not None else {}
+        no_proxy = proxies.get('no_proxy') if proxies is not None else None
 
-        if self.trust_env and not should_bypass_proxies(url):
-            environ_proxies = get_environ_proxies(url)
+        if self.trust_env and not should_bypass_proxies(
+                url, no_proxy=no_proxy):
```

Comment:
Rather than break this line, which is kind of ugly, let's just create a named variable (maybe `bypass`) that we assign the value of this to so we can keep the conditional clear.